internal/resources/google/compute_cost_component_helpers.go

```diff
@@ -202,6 +204,37 @@ func computeCostComponents(region, machineType string, purchaseOption string, in
 
 }
 
+func getSustainedUseDiscount(monthlyHours *float64, thresholds []float64, rates []float64, incrementalRates []float64) float64 {
+	// Assume 730 hours(max) if monthly_hrs is not set
+	if *monthlyHours == 0 {
+		*monthlyHours = 730.0
+	}
+	totalHoursInMonth := 730.0
```

Comment:
```suggestion
func getSustainedUseDiscount(monthlyHours float64, thresholds []float64, rates []float64, incrementalRates []float64) float64 {
```

If we move the logic to fallback to `schema.HourToMonthUnitMultiplier` above, then we shouldn't need this and we can pass in the hours as a non-pointer float64. If monthlyHours is 0, we should still support that.